components/Form.js

```diff
@@ -16,57 +16,66 @@ export default function Form({ onSubmit, name, placeholder }) {
         name={name}
         placeholder={placeholder}
         onChange={(event) => setCount(event.target.value.length)}
+        autoComplete="off"
         required
       />
       <CounterWrapper>
         <Counter maxLength={30} counter={count} />
       </CounterWrapper>
-      <SubmitButton type="submit">+</SubmitButton>
+      <SubmitButton type="submit">add</SubmitButton>
     </StyledForm>
   );
 }
 
 const StyledForm = styled.form`
   display: flex;
   position: fixed;
-  bottom: 5px;
+  bottom: 3.5em;
   width: 100%;
   flex-direction: column;
-  align-items: center;
   margin: 0;
   padding: 0;
 `;
 
 const Input = styled.input`
-  width: 85%;
-  margin: 5px;
-  padding: 10px;
+  position: absolute;
```

Comment:
I set the ```position: absolute``` because otherwise I kind of can't set the left and right property of the input and submit button. Is there an other way I did overlook?

Reply:
I just wanted to note that you can apply all your stylings with simpler ways and you do not need to set position: absolute all the time. :) Instead of working with positioning you could use a padding-left in your `StyledForm` for example.